Algebras/DotBracket/alg_dotBracket.gap

```diff
@@ -1,4 +1,12 @@
 algebra alg_dotBracket implements sig_foldrna(alphabet = char, answer = string) {
 	include "Algebras/DotBracket/Parts/algpart_dotBracket_basic.gap"
+	include "Algebras/DotBracket/Parts/algpart_cofold_dotBracket.gap"
 	include "Algebras/DotBracket/Parts/algpart_dotBracket_macrostate.gap"
 }
+
+algebra alg_dotBracket_unique extends alg_dotBracket {
+	include "Algebras/DotBracket/Parts/algpart_dotBracket_unique.gap"
```

Comment:
I think we can remove this additional file completely, bet let me know your incentive for adding it. I might overlook something here.